### ISSUE TITLE
Bug: Paypal confirmation page not displaying after purchase

### ISSUE THREAD

**Issue by zdxn** [zdxn/ValkyriesFall#12]
Issue:
After making a purchase, there is no confirmation page that indicates that your purchase was successful or not.

Resolution:
Clearly indicate that purchases have completed.

Urgency:
5

**Comment by DavePearcey** [comment]
Added in alert, like the failure and cancellation messages.
